apps/frontend/src/shared/i18n/locales/en/dialogs.json

```
@@ -22,38 +22,118 @@
+    "branchHelp": "All tasks will be created from branches like",
+    "basedOn": "based on",
     "whyBranch": "Why select a branch?",
     "branchExplanation": "Auto Claude creates isolated workspaces for each task. Selecting the right base branch ensures your tasks start with the latest code from your main development line.",
-    "ready": "Auto Claude is ready to use! You can now create tasks that will be automatically based on the {{branchName}} branch."
+    "completeSetup": "Complete Setup",
+    "setupComplete": "Setup Complete",
+    "ready": "Auto Claude is ready to use! You can now create tasks that will be automatically based on the {{branchName}} branch.",
+    "authenticate": "Authenticate",
+    "configure": "Configure"
   },
   "worktrees": {
```

Biome (2.1.2)

[error] The key worktrees was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)